src/openforms/registrations/contrib/objects_api/api.py

```diff
@@ -0,0 +1,79 @@
+from typing import Any
+
+from django.urls import path
+from django.utils.translation import gettext_lazy as _
+
+from drf_spectacular.utils import extend_schema, extend_schema_view
+from rest_framework import authentication, permissions, serializers, views
+
+from openforms.api.views import ListMixin
+
+from .client import get_objecttypes_client
+
+
+class ObjecttypeSerializer(serializers.Serializer):
+    # Keys are defined in camel case as this is what we get from the Objecttype API
+    url = serializers.URLField(
+        label=_(
+            "URL reference to this object. This is the unique identification and location of this object."
+        ),
+    )
+    uuid = serializers.UUIDField(label=_("Unique identifier (UUID4)."))
+    name = serializers.CharField(label=_("Name of the object type."))
+    namePlural = serializers.CharField(label=_("Plural name of the object type."))
+
+
+class ObjecttypeVersionSerializer(serializers.Serializer):
+    version = serializers.IntegerField(
+        label=_("Integer version of the Objecttype."),
+    )
```

Comment:
This currently would return a list with objects `{"version": "X"}`. I wonder if it would be nice for the endpoint to return a list of versions directly (without the objects) :thinking:

Reply:
Yes probably better. I'm also wondering if it would make sense to have a single endpoint. I don't have any preference, in the frontend it's mostly a matter of doing a request vs `.filter` coupled with `useEffect`

Reply:
Turns out a single API endpoint might not be a good idea: I would have to make `1 + n` requests for `n` objecttypes

Reply:
keep in mind the scale of data - it's reasonable to assume tens/hundreds of objecttypes and versions will exist at some point in time (say 100 objecttypes with 20 versions each -> 2000 records returned).

Holding all that data in memory on the client-side is not optimal.

Reply:
w/r to the `version` key - having a list of objects allows for future changes in the API to add additional metadata. returning a list of strings will require you to fight DRF and offers less flexibility, so I'd stick with the explicit serializer.